x/stake/keeper.go

```diff
@@ -354,7 +354,7 @@ func (k Keeper) clearAccUpdateValidators(ctx sdk.Context) {
 
 //_____________________________________________________________________
 
-// load a delegator bong
```

Comment:
you didn't like my joke?

Reply:
Loved it, just didn't realize it was intentional.